src/Proto.Cluster.Extensions/HostedClusterService.cs

```diff
@@ -0,0 +1,50 @@
+﻿// -----------------------------------------------------------------------
```

Comment:
It's already in contrib for a couple of months https://github.com/AsynkronIT/protoactor-dotnet-contrib/pull/2/files#diff-05d243082ea68e8c49700f5bc7fc94d2